src/main/java/com/senecafoundation/CharacterSelection.java

```diff
@@ -0,0 +1,182 @@
+package com.senecafoundation;
+import com.senecafoundation.DataHandler.IDataHandler;
+
+
+import java.util.Scanner;
+
+public class CharacterSelection 
+{
+    //Variables
+    private Scanner input = new Scanner(System.in);
+    private IDataHandler dataHandler;
+
+    //Constructor
+    public CharacterSelection()
+    {
+        
+    }
+
+
+    //Methods
+
+    public String pickName()
+    {
+        System.out.println("Character Name: \n");
+        String characterName = input.nextLine();
+        
+        return characterName;
+    }
+
+    public int pickAge()
+    {
+
+        int characterAge = 0;
+
+        while(characterAge <= 0 || characterAge >= 100)
+        {
+            System.out.println("Character Age: \n");
+            int characterAgeDummy = input.nextInt();
+
+            characterAge = characterAgeDummy;
+
+            if(characterAgeDummy <= 0 || characterAgeDummy >= 100)
+            {
+                System.out.println("Error!"); 
+            }
+
+        }
+
+        return characterAge;
+    }
+
+    public String pickSex()
+    {
+        System.out.println("Character Sex: Male/Female \n");
+        String characterSex = input.nextLine();
+
+        if((characterSex.equals("Male")) || (characterSex.equals("male")))
+        {
+            characterSex = "Male";
+        }
+        else if((characterSex.equals("Female")) || (characterSex.equals("female")))
+        {
+            characterSex = "Female";
+        }
+        else
+        {
+            characterSex = "Unknown";
+        }
+
+        return characterSex;
+    }
+
+
+
+    public Character picker() throws Exception
+    {
+        Character character;
+        boolean flag = true;
+
+        while(flag)
+        {
+
+            System.out.println("\nPick A Character: \n Orc, Viking, DemiHuman, Elf, Nuet, Halfling, HalfElf, ShadowElf, Human, Tiefler");
+            String race = input.nextLine();
+
+            if((race.equals("Orc")) || (race.equals("orc")))
+            {
+                character = new Orc("",0,"",25,20,dataHandler);
+                character.setName(pickName());
```

Comment:
Can we do something better with these five lines?  So they aren't copy and paste code?  That's usually a sign of stinky code.